CONTRIBUTING.md

```diff
@@ -1,256 +1,339 @@
+One or more maintainers of the maintainers listed in the `owners` element should be equal to a similar name listed in our `CODE_OWNERS` file, and will be considered the main maintainer of the package.
 
-Since commit message is used to push packages to the community repository (CR), prior to commit of the AU package specify correct `NuspecVersion` in the nuspec file as follows:
+#### Icons
 
-1. If you want to fix the exisiting CR package version for which no update is available at vendor specify the `NuspecVersion` the same as `RemoteVersion`.
-1. If you want to push new package version to CR and `RemoteVersion` doesn't exist on CR, set `NuspecVersion` to `0.0`. AU updater will then bring the version to correct one during the next AU run and commit it to the git repository.
-1. If you want to fix package that is pushed but not yet approved (for example it fails verifier) do the same as in first case but request in PR comment that you want version explicitelly set.
+A package must have an icon available if the packaged software has an icon. This icon must be named the same as the package and is placed in the [icons](https://github.com/chocolatey-community/chocolatey-packages/tree/master/icons) directory.
+If the package name ends with either `.install`, `.portable` those suffixes may be ignored in the icon name.
+When an icon is added to this folder with the correct name, it wwill **automatically** be set in the metadata file and the Readme file when our build server updates the package if the metadata file contains an `<IconUrl>` element.
 
-__NOTE__: Automatic fix version doesn't work if package is using the _revision_ part of the version (4th number). In that case _explicit version_ must be used: `[AU package:version]`.
-## 1.4 UI Automation
+**IMPORTANT: If there is no icon available, the comment `<!-- IconUrl: Skip check -->` is required to be specified in the metadata file**
+
+#### Chocolatey compatibility
+
+All packages are expected to be compatible with the version of Chocolatey CLI that was available 12 months earlier than the last change of the package.
+With that, it means that if you make a change to the nuspec, or one of the package scripts on January 12th, 2023 it should be compatible with Chocolatey CLI 0.11.3 and later released versions.
+If that is not possible, and the package needs a version of Chocolatey CLI that was released later, it will need to include a dependency for that Chocolatey CLI version.
+
+### Chocolatey Scripts
+
+All PowerShell scripts are expected to be compatible with PowerShell v2 and later (not PowerShel Core). If this is not possible, a dependency on the package `PowerShell` is required with its minimum version set to the earliest version that the scripts in the package is expected to work with.
+
+**TODO: This section needs to be updated, and is a copy-paste from earlier guidelines**
```

Comment:
Just flagging this, either for removal (if the below is alright) or action.

Reply:
Yeah, I think it can be removed

```suggestion
```